### DISCUSSION TITLE
On `BeatmapInfoWedge`, beatmaps with very long Source shouldn't have their Song Title be significantly omitted

### DISCUSSION THREAD

**Discussion by Nivalyx** [ppy/osu#15543] [Q&A] [answered]
Noticed this issue on certain Ranked/Qualified beatmaps which has a super long Source to their name, such as http://osu.ppy.sh/s/576489 and http://osu.ppy.sh/s/1596341 for example :

![image](https://user-images.githubusercontent.com/36564236/140976186-c5aabe3d-5318-4d5a-bdf4-48e69a4a6ad3.png)

![image](https://user-images.githubusercontent.com/36564236/140973003-23b68c55-8d7a-415b-a0e6-a4d70c3a7d47.png)

---

I have no idea whether this behaviour is intentional or not tho, but i feel like separating the beatmap's Source and Title in the infobox altogether into something like this would be much better both visually and informatively (esp. on cases where the beatmap has a super long Source as mentioned above) :

![image](https://user-images.githubusercontent.com/36564236/140976803-1ad60620-721e-4ee9-b472-d736fad6bc66.png)

**Comment by bdach** [comment, marked as answer]
The song select screen is pending a redesign anyway, at which point this will likely be addressed. See: https://www.figma.com/file/DXKwqZhD5yyb1igc3mKo1P/Song-Select-Draft-2?node-id=0%3A1
